Permanent/Modules/Invitations/Views/InvitesView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  InvitesView.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 06.02.2024.
+
+import SwiftUI
+import UIKit
+
+struct InvitesView: View {
```

Comment:
See if there is a way to use a single ViewRepresentableContainer and just have two vars:

var viewRepresentable: UIViewControllerRepresentable
 var title: String